### PR TITLE
fix build badge

### LINKED ISSUE #20: missing changelog.md file
Would be nice to have changelog.md file so it also shows up in the community supermarket page for this cookbook.

### PR DESCRIPTION
**Pull request by Xorima**
## Description

[Describe what this change achieves]

### Issues Resolved

#20 

### Check List
- [ ] All tests pass. See https://github.com/sous-chefs/apache2/blob/master/TESTING.md
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by Stromweld** [comment]
@Xorima can you add a changelog.md file to this repo #20

**Comment by Xorima** [comment]
Changelog added

File: chefignore [hunk 1]
```diff
@@ -90,7 +90,6 @@ Policyfile.lock.json
 # Cookbooks #
 #############
 CONTRIBUTING*
-CHANGELOG*
```
Comment (damacus) [inline chefignore]: changelog isn't in the chefignore anymore? 🤔

Comment (Xorima) [reply chefignore]: See #20 was requested this way (not that stove seems to take chefignore into account ...)

Comment (Stromweld) [reply chefignore]: sorry about that, I could of sworn we had issues a while back with changelogs not getting pushed to our internal supermarket without taking out that line in the chefignore. Granted we were using knife supermarket share and not stove.

**Pull request merged**